src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -152,7 +152,7 @@ public async Task<ActionResult> DeletePolicy([FromBody] RequestToDeletePolicyLis
                 return StatusCode(206, deletionResults);
             }
 
-            _logger.LogInformation($"Deletion could not be completed. None of the rules could be processed, indicating invalid or incomplete input:\n{policiesToDeleteSerialized}", policiesToDeleteSerialized);
+            _logger.LogInformation("Deletion could not be completed. None of the rules could be processed, indicating invalid or incomplete input:\n{policiesToDeleteSerialized}", policiesToDeleteSerialized);
```

Comment:
The same string interpolation issue exists on line 114  and 151

Reply:
Those does not fail as the interpolation is done on variables not in the log statement itself but removed them as well